addon/components/sidebar.hbs

```diff
@@ -0,0 +1,54 @@
+<div class='say-editor-hints'>
+  {{#if (has-block "collapsible")}}
+    <div class='say-editor-hints__insert'>
+      <div class='au-c-accordion au-c-accordion--reverse au-c-accordion--border'>
+        <AuToolbar @nowrap={{true}} @reverse={{true}} {{on 'click' this.toggle}}as |Group|
+        >
+          <Group>
+            <div>
+              <AuButton
+                @skin='tertiary'
+                aria-hidden='true'
+                focusable='false'
+                aria-expanded='{{if this.expanded "true" "false"}}'
+              >
+                {{t "ember-rdfa-editor.insert"}}
+              </AuButton>
+            </div>
+          </Group>
+          <Group>
+            {{#if this.expanded}}
+              <AuIcon
+                @icon='remove'
+                @alignment='left'
+                @size='large'
+                @ariaHidden='true'
+              />
+              <span class='au-u-hidden-visually'>
+                Open accordion
+              </span>
+            {{else}}
+              <AuIcon
+                @icon='add'
+                @alignment='left'
+                @size='large'
+                @ariaHidden='true'
+              />
+              <span class='au-u-hidden-visually'>
+                Sluit accordion
+              </span>
+            {{/if}}
+          </Group>
+        </AuToolbar>
+        <div class={{if this.expanded 'au-c-accordion__content' 'au-u-hidden'}}>
+          <AuList @divider={{true}}>
+            {{yield to="collapsible"}}
```

Comment:
I'd yield this accordion list as a contextual component here, so you can have multiple collapsible sections

Reply:
Good idea, I'm wondering if we convert all yielded named blocks to contextual components to make it consistent.